tests/ssl.py

```diff
@@ -11,9 +11,19 @@
     ssl = SslApp(cluster)
 
     a = ssl.create_keystore('mybroker')
-    print('created keystore: %s' % a)
+    print('created keystore: %s' % (a,))
```

Comment:
Guess this was not intended?

Reply:
Actually this is necessary. I don't know if this ever worked (maybe `create_keystore` used to have a different signature, or this might have worked in python2?) but this was throwing a `TypeError: not all arguments converted during string formatting` exception. This is because the return value of `create_keystore` is a tuple, and the `%` operator is trying to format the string with the elements in the tuple, rather than simply printing the entire tuple out with a `%s`.

`(a,)` is simply the (very awkward) syntax for a tuple which contains a single element, which is itself a tuple here.

Reply:
Ah, makes sense :+1: